pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -586,7 +589,105 @@ func TestZeroHintMath(t *testing.T) {
 				}),
 			},
 		},
-
+		"SignedDivRem": {
+			{
+				operanders: []*hintOperander{
+					{Name: "value", Kind: apRelative, Value: feltString("0")},
```

Comment:
I would rather use `feltString()` for bigger values, those you can't represent as uint64 for example. Passing the value as a string surely makes the process of getting the felt less efficient. You can use `feltUint64` or `feltInt64` instead or direct values from the `constant` file. In this case you could directly use `FeltZero`

Check the rest of the felts you passed in these tests because this comment applies to most of them

Reply:
Applied that to the tests